dependencies/jessie/foreman-tasks-core/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-foreman-tasks-core (0.1.1-1) stable; urgency=low
```

Comment:
As said, there's nothing we can do about this here... per convention "ruby-foreman-*" are Foreman core plugins, but using the convention for "standard" Debian Ruby packages (what we put under "dependencies" also leads to such a name now).